adyen.go

```diff
@@ -97,13 +103,13 @@ func (a *Adyen) SetCurrency(currency string) {
 }
 
 // execute request on Adyen side, transforms "requestEntity" into JSON representation
-func (a *Adyen) execute(method string, requestEntity interface{}) (*Response, error) {
+func (a *Adyen) execute(method string, service string, requestEntity interface{}) (*Response, error) {
```

Comment:
Maybe for consistency, we should change an order of params? now it's `method, service` but adyenURL accept `service, method`. I think `service, method` make more sense, cause we are moving from generic to the specific method that needs to be called.

Reply:
No problem, I'll push an update